File: official/docs/ruby/carrier_accounts/update_carrier_account.rb
require 'easypost'

EasyPost.api_key = ENV['EASYPOST_API_KEY']

carrier_account = EasyPost::CarrierAccount.retrieve('ca_...')

carrier_account.description = 'FL Location UPS Account'
carrier_account.credentials['pickup_id'] = 'abc123'

carrier_account.save

puts carrier_account

File: official/docs/node/carrier_accounts/update_carrier_account.js
const Easypost = require('@easypost/api');

const api = new Easypost(process.env.EASYPOST_API_KEY);

const carrierAccount = api.CarrierAccount.retrieve('ca_...');

carrierAccount.credentials.pickup_id = 'abc123';

carrierAccount.save().then(console.log);
